dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201016UpdatePrimaryKeyLengthIdentifierTable.java

```diff
@@ -0,0 +1,66 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import com.dotmarketing.util.Logger;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class Task201016UpdatePrimaryKeyLengthIdentifierTable extends AbstractJDBCStartupTask {
+
+    @Override
+    public boolean forceRun() {
+        final Connection connection = DbConnectionFactory.getConnection();
```

Comment:
Codacy found an issue: [Ensure that resources like this Connection object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354748)